MediaBrowser.Controller/Sorting/SortExtensions.cs

```diff
@@ -30,5 +30,15 @@ public static IOrderedEnumerable<T> ThenByStringDescending<T>(this IOrderedEnume
         {
             return list.ThenByDescending(getName, _comparer);
         }
+
+        public static string Trimmed(this string? value)
```

Comment:
This one gets used when we have a possibly-null `string`

Reply:
I think I would prefer to just do `value?.Trim()` instead of having a function that turns null into an empty string.

Reply:
That's fine. Excepting there's a lot of places where this is used in an OrderBy or ThenBy and was coalescing to a `string.Empty` or `""`. You make the call for what you want, I can rip through to swap it.

Reply:
I've swapped over to `?.Trim()`